### DISCUSSION TITLE
[Feature request] Include Python 3.9 by default or optionaly as addon

### DISCUSSION THREAD

**Discussion by r57zone** [OnionUI/Onion#1452] [Ideas]
Dear devs, it would be cool to add Python 3.9 by default. This would be great and convenient for homebrew development.
https://github.com/OnionUI/Onion/discussions/1034#discussioncomment-7527471
Thanks

**Comment by schmurtzm** [comment]
Hi, the problem is that the pythion 3.9 is bigger than Onion entirely 😅
So it's not an option. However it could be documented in official Onion documentation 🤔

**Comment by r57zone** [comment]
Maybe it would be possible to make some kind of external addon ? which could be downloaded and unzipped.

Usually use this device with a large memory card and memory is not a problem. Python 3 will allow we to use it not only for games, but also for various custom software.
